tests/function_modifiers/test_recursive.py

```diff
@@ -3,22 +3,25 @@
 import random
 from typing import Tuple
 
+import pandas as pd
 import pytest
 
-from hamilton import ad_hoc_utils, graph
+from hamilton import ad_hoc_utils, driver, graph, node
```

Comment:
Could use

```python
from hamilton import ..., node as hamilton_node
```

because the name `node` is overriden elsewhere (e.g., line 187)